guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -446,19 +453,19 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
                         noDirectUpdate={true}
                         onSelect={(value: any) => {
                             this._sizeOption = value;
-                            if (this._sizeOption != (sizeOptions.length - 1)) {
+                            if (this._sizeOption != (sizeOptions.length)) {

```

Comment:
```suggestion
                            if (this._sizeOption !== (sizeOptions.length)) {
```

Always use `===` or `!==`, unless it is really needed to compare different types. It's less of a performance issue and more of a runtime-error-handling thing. this should never work if one of those types is different than the other.